packages/react-components/src/organisms/SharedResearchOutputBanners.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react';
+import { ResearchOutputResponse } from '@asap-hub/model';
+import Toast from './Toast';
+
+export interface SharedResearchOutputBannersProps {
+  association: string;
+  documentType: string;
+  published: boolean;
+  reviewRequestedBy: ResearchOutputResponse['reviewRequestedBy'];
+  publishedNow: boolean;
+  draftCreated?: boolean;
+  reviewToggled: boolean;
+  associationName: string;
+}
+
+const SharedResearchOutputBanners: React.FC<
+  SharedResearchOutputBannersProps
+> = ({
+  association,
+  documentType,
+  published,
+  publishedNow,
+  reviewRequestedBy,
+  draftCreated,
+  reviewToggled,
+  associationName,
+}) => {
+  const [publishedNowBanner, setPublishedNowBanner] = useState(published);
+  const [draftCreatedBanner, setDraftCreatedBanner] = useState(draftCreated);
+  const [reviewBannerState, setReviewBannerState] = useState(
+    reviewToggled ? (reviewRequestedBy ? 'requested' : 'dismissed') : null,
+  );
+
+  useEffect(() => {
+    setReviewBannerState(
+      reviewToggled ? (reviewRequestedBy ? 'requested' : 'dismissed') : null,
+    );
+  }, [reviewToggled, reviewRequestedBy]);
+
+  return (
+    <>
+      {draftCreatedBanner && (
+        <Toast
+          accent="successLarge"
+          onClose={() => setDraftCreatedBanner(false)}
+        >
+          {`Draft ${
+            association === 'working group' ? 'Working Group' : 'Team'
+          } ${documentType} created successfully.`}
+        </Toast>
+      )}
+      {reviewBannerState === 'requested' && (
+        <Toast accent="successLarge" onClose={() => setReviewBannerState(null)}>
+          {`Draft ${association} ${documentType} submitted for PM review successfully.`}
+        </Toast>
+      )}
+      {reviewBannerState === 'dismissed' && (
+        <Toast accent="successLarge" onClose={() => setReviewBannerState(null)}>
+          {`In review ${association} ${documentType} switched to draft successfully.`}
+        </Toast>
+      )}
+
+      {reviewRequestedBy && !published && (
+        <Toast accent="info">
```

Comment:
it seems that the color of the `info` accent does not match figma, the text should be `info500` as well the same way it is in the tag. I'm not sure if it should change to all the places. It would be better to talk to Manuel later about it :)

Reply:
I'll discuss this with Manuel